### DISCUSSION TITLE
Icon Request: fa-balcony

### DISCUSSION THREAD

**Discussion by Mohmdthabet** [FortAwesome/Font-Awesome#21178] [Icon Requests]
balcony icon 
![balcony](https://user-images.githubusercontent.com/9333195/28893614-6dc8bbe4-77d2-11e7-8089-03159b85ba08.png)

this icon refers to balcony which found in an apartment.

**Comment by sdkiko** [comment]
**PRETTY PLEASE MR.FONTAWESOME**

**Comment by Attrexx** [comment]
we need real estate icons indeed
+1

**Comment by hxlyhxly** [comment]
+1

**Comment by Smexhy** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
